src/ansys/meshing/prime/graphics/plotter.py

```diff
@@ -0,0 +1,353 @@
+        """Add a scope to the plotter.
+
+        Parameters
+        ----------
+        model : Model
+            Model to add to the plotter.
+        scope : str
+            Scope to add to the plotter.
+        """
+        model_pd = model.get_scoped_polydata(scope)
+        self.add_model_pd(model_pd)
+
+    def plot_iter(
+        self,
+        plotting_list: List[Any],
+        name_filter: str = None,
+        **plotting_options,
+    ) -> None:
+        """
+        Add a list of any type of object to the scene.
+
+        Allowed types are PyPrime models or any PyVista plottable object.
+
+        Parameters
+        ----------
+        plotting_list : List[Any]
+            List of objects you want to plot.
+        name_filter : str, default: None
+            Regular expression with the desired name or names you want to include in the plotter.
+        **plotting_options : dict, default: None
+            Keyword arguments. For allowable keyword arguments, see the
+            :meth:`Plotter.add_mesh <pyvista.Plotter.add_mesh>` method.
+        """
+        for plottable_object in plotting_list:
+            _ = self.plot(plottable_object, name_filter, **plotting_options)
+
+    def plot(
+        self,
+        plottable_object: Any,
+        scope: prime.ScopeDefinition = None,
+        name_filter: str = None,
+        **plotting_options,
+    ):
+        """Add an object to the plotter.
+
+        Allowed types are PyPrime models or any PyVista plottable object.
+
+        Parameters
+        ----------
+        plottable_object : Any
+            Object to add to the plotter.
+        scope : prime.ScopeDefinition, optional
+            Scope you want to plot, by default None.
+        name_filter : str, optional
+            Regular expression with the desired name or names you want to include in the plotter,
+            by default None.
+        """
+        if isinstance(plottable_object, Model):
+            self.add_model(plottable_object, scope)
+        elif isinstance(plottable_object, List):
+            self.plot_iter(plottable_object, name_filter, **plotting_options)
+        else:
+            self._backend.pv_interface.plot(plottable_object, name_filter, **plotting_options)
+
+    def show(
+        self,
+        plottable_object: Any = None,
+        screenshot: str = None,
+        name_filter: bool = None,
+        scope: prime.ScopeDefinition = None,
+        **plotting_options,
+    ) -> None:
+        """Show the plotted objects.
+
+        Parameters
+        ----------
+        plottable_object : Any, optional
+            Object to show, by default None.
+        screenshot : str, optional
+            Path to save a screenshot, by default None.
+        name_filter : bool, optional
+            Flag to filter the object, by default None.
+        plotting_options : dict
+            Additional plotting options the selected backend accepts.
```

Comment:
```suggestion
        plottable_object : Any, default: None
            Object to show.
        screenshot : str, default: None
            Path to save a screenshot to.
        name_filter : bool, default: None
            Whether to filter the object.
        plotting_options : dict, default: None
            Additional plotting options that the selected backend accepts.
```
Is ``name_filter`` really a Boolean? It seems like it would be string?

Reply:
This is a mistake, it is indeed an string.